report_dynamic/views/report_dynamic.xml

```diff
@@ -0,0 +1,200 @@
+                            'hover_true': 'PAUSE',
+                            'string_false': 'HELP',
+                            'hover_false': 'FINISH'
+                        }}"
+                            />
+                        </button>
+                        <field name="section_ids" invisible="1" />
+                        <button
+                            type="object"
+                            name="action_view_section"
+                            class="oe_stat_button"
+                            icon="fa-file-text-o"
+                            attrs="{'invisible': [('section_count', '=', 0)]}"
+                            groups="base.group_user"
+                        >
+                            <field
+                                name="section_count"
+                                widget="statinfo"
+                                string="Sections"
+                            />
+                        </button>
+                    </div>
+                    <div class="oe_title">
+                        <label for="name" class="oe_edit_only" string="Report name" />
+                        <h1>
+                            <field name="name" />
+                        </h1>
+                        <h2 attrs="{'invisible': [('is_template', '=', False)]}">
+                            <label for="model_name" />
+                            <field name="model_name" />
+                        </h2>
+                    </div>
+                    <group name="main">
+                        <group name="left">
+                            <field
+                                name="model_id"
+                                attrs="{'invisible':[('is_template', '=', False)]}"
+                            />
+                            <label for="resource_ref" />
+                            <div>
+                                <field
+                                    name="resource_ref"
+                                    options="{'hide_model': True, 'no_create': True, 'no_edit': True, 'no_open': True}"
+                                    class="oe_inline"
+                                />
+                                <!-- Make it obvious that target record for templates is for previewing -->
+                                <span
+                                    attrs="{'invisible':['|', ('is_template', '=', False),('model_id', '=', False)]}"
+                                >
+                                    <strong> PREVIEW</strong>
+                                </span>
+                            </div>
+                            <field
+                                name="condition_domain_global"
+                                widget="domain"
+                                options="{'model': 'report.dynamic', 'in_dialog': True}"
+                                attrs="{'invisible': [('is_template', '=', False)]}"
```

Comment:
Add a helptext here:

`help="When you have multiple templates for one model (eg. a contract for senior employees and another for juniors), this domain is used to determine the domain condition that decides for which records to choose this template (eg. [('employee_id.skill_level', '=', 'junior')])`